pages/profile-setup/index.page.tsx

```diff
@@ -1,3 +1,56 @@
-export default function ProfileSetup() {
-  return <div>프로필 입력 페이지입니다.</div>;
+import { GetServerSidePropsContext } from "next";
+
+import Image from "next/image";
+
+import classNames from "classnames/bind";
+
+import ProfileSetupForm from "./components/ProfileSetupForm/ProfileSetupForm";
+import { instance } from "@/apis/axios";
+
+import styles from "./index.module.scss";
+
+const cn = classNames.bind(styles);
+
+interface ProfileSetupProps {
+  accountId: number;
+  email: string;
+}
+
+export const getServerSideProps = async (context: GetServerSidePropsContext) => {
+  const accountId = context.query.id;
+
+  const res = await instance.get(`/account-info/${accountId}`);
+  const { email } = res.data;
+
+  return {
+    props: { accountId, email },
+  };
+};
+
+export default function ProfileSetup({ accountId, email }: ProfileSetupProps) {
+  return (
+    <>
+      <header className={cn("header")}>
+        <Image src="/icons/tickita-logo.svg" alt="logo" width={60} height={22} />
```

Comment:
```suggestion
        <Image src="/icons/tickita-logo.svg" alt="티키타 로고" width={60} height={22} />
```
p3: `alt`는 한국어로 적는게 좋을 것 같아요~

Reply:
지난번에도 한 번 피드백 주셨던 부분인데 또 영어로 써놨군요 제가... 반성합니다... 😢